src/net/udp.rs

```diff
@@ -85,6 +102,38 @@ mod test {
     }
 
     #[test]
+    fn send_receive_fragment_packet() {
+        let mut send_socket = UdpSocket::bind("127.0.0.1:12347").unwrap();
+        let mut recv_socket = UdpSocket::bind("127.0.0.1:12348").unwrap();
+
+        let addr = SocketAddr::new(
```

Comment:
btw its easier to create a `SocketAddr` via doing `"127.0.0.1:12348".parse().unwrap()`